lib/main.dart

```diff
@@ -23,9 +25,35 @@ void main() async {
   environment = configuration.getEnvironment();
   SimpleLogger().info("Running in $environment environment");
 
+  SimpleLogger().info("Connecting to MQTT broker...");
+
+  await _initializeMqttClient();
+
   runApp(const MyApp());
 }
 
+void _initPeriodicStatusMessage() {
+  Timer.periodic(const Duration(seconds: 30), (_) {
+    if (mqttClient.isConnected) {
+      mqttClient.sendStatusMessage(true);
+    }
+  });
+}
+
+Future<void> _initializeMqttClient() async {
+  final String? serialNumber = await DeviceInfo.getSerialNumber();
+
+  if (serialNumber == null) {
+    SimpleLogger().warning("Device ID not found, cannot connect to MQTT.");
```

Comment:
Device ID does sound a bit misleading as it seems to refer to a Device within Noheva which has not been created at this point yet.

However this is probably subject to change once API support for Devices is ready